src/frontend/efiling-frontend/src/components/page/payment/__snapshots__/Payment.test.js.snap

```diff
@@ -378,6 +378,156 @@ you feel are necessary.
         </button>
       </section>
     </div>
+    <div
+      class="sidecard"
+    >
+      <div
+        class="wide-dashboard-spacing"
+        style="position: relative;"
+      >
+        <div
+          class="col-12 pt-2"
+          style="position: relative;"
+        >
+          <p />
+          <section
+            class="bluegrey-container"
+            id="bluegrey-section"
+          >
+            <div
+              class="container-background bluegrey-heading"
+            >
+              <h2
+                class="heading-style"
+              >
+                <div
+                  class="side-card-row"
+                >
+                  <div
+                    class="round-icon-wrapper"
+                  >
+                    <svg
+                      class="side-card-icon"
+                      fill="currentColor"
+                      height="1em"
+                      stroke="currentColor"
+                      stroke-width="0"
+                      viewBox="0 0 24 24"
+                      width="1em"
+                      xmlns="http://www.w3.org/2000/svg"
+                    >
+                      <path
+                        d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
+                      />
+                    </svg>
+                  </div>
+                  <div
+                    class="side-card-title"
+                  >
+                    Your CSO Account
+                  </div>
+                </div>
+              </h2>
+            </div>
+            <div
+              class="bluegrey-content"
+            >
+              <span
+                class="content-style"
+              >
+                <p>
+                  CSO account 
+                  <b />
+                   is linked to your Basic BCeID account 
+                  <b>
```

Comment:
isn't strong better than bold? I remember being told this once for some reason.

Reply:
yes actually you're right!

<strong> means "strong emphasis", and implies no particular visual style. It has semantic meaning, but could look like anything. <b> is used to apply a bold visual effect to text, but is a presentational tag like <font> and so should be avoided (where possible) in favour of CSS